Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManagerProfile.cs

```diff
@@ -101,7 +92,7 @@ public static MRTKOculusConfig Instance
         /// <summary>
         /// Event triggered when the custom material for hand mesh is updated.
         /// </summary>
-        public static System.Action OnCustomHandMaterialUpdate;
+        public System.Action OnCustomHandMaterialUpdate;
```

Comment:
We might want to change the pattern here, since we've tried to hold off on the pattern of changing the profile at runtime. Could the hand material be set on the device manager instead (or really, I feel like it should be applied directly to the rigged hands instead of asking this class to do it)?

Reply:
This one's maybe a larger undertaking (unless we just remove this ability (for now?)), and since it's already present we could hold off on making this change during this initial round

Reply:
I think the prefabs are preconfigured with the appropriate material so I can just remove it

Reply:
Actually passing on this since we're using a custom material here to have them react to the pinch strength, removing this makes it harder to ensure the pinch strength property exists on the hand material and can result in shader errors, so will leave it be for this round.